mctx/_src/policies.py

```diff
@@ -309,9 +307,9 @@ def stochastic_muzero_policy(
   # construct a dummy afterstate embedding
   batch_size = jax.tree_util.tree_leaves(root.embedding)[0].shape[0]
   dummy_action = jnp.zeros([batch_size], dtype=jnp.int32)
-  _, dummy_afterstate_embedding = decision_recurrent_fn(params, rng_key,
-                                                        dummy_action,
-                                                        root.embedding)
+  dummy_output, dummy_afterstate_embedding = decision_recurrent_fn(
+    params, rng_key, dummy_action, root.embedding)
```

Comment:
Here, the identation should be 4 spaces, because it is a "hanging" indent.
Thanks.